src/metadata/taglib_handler.cc

```diff
@@ -466,12 +466,20 @@ void TagLibHandler::extractMP3(TagLib::IOStream* roStream, const std::shared_ptr
                 const auto textFrame = dynamic_cast<const TagLib::ID3v2::TextIdentificationFrame*>(frame);
                 if (textFrame == nullptr)
                     continue;
-                const TagLib::String frameContents = textFrame->toString();
-                std::string value(frameContents.toCString(true));
+                std::string content = "";
+                std::string subTag = "";
+                for (auto&& item : textFrame->fieldList()) {
```

Comment:
My fault, but may you rename `item` to `field`. `item` is already used for the `CdsItem` argument.

Reply:
Fixed (and also fixed other logical failure in IF statement)